ams/test_data/data.py

```diff
@@ -3,14 +3,27 @@
 from .. import db
 from ..models.User import User
 from ..models.Staff import Staff
+from ..models.Student import Student
 
 
 def seed_data():
     
     password = 'password'
     user_password = generate_password_hash(password, 'sha256')  
     user1 = User(username='francis', email='francis@bolton.ac.uk', telephone='78124523', password=user_password, is_staff=True)
+    user2 = User(username='ibtisam', email='ibtisam@bolton.ac.uk', telephone='78124524', password='password', is_staff=True)
```

Comment:
you need to hash the password.
**_generate_password_hash(password, 'sha256')_**
as i did in line 12